README.md

```diff
@@ -39,7 +39,15 @@ Start Agent with:
 ```bash
 MF_AGENT_BOOTSTRAP_ID=<bootstrap_id> \
 MF_AGENT_BOOTSTRAP_KEY=<bootstrap_key> \
-MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/bs/things/bootstrap \
+MF_AGENT_BOOTSTRAP_URL=https://mainflux.com/things/bootstrap \
```

Comment:
this must be `http://mainflux.com:8204/things/bootstrap` if `bootstrap` is started additionally to the default docker composition ( and there is no mapping in nginx.conf for the bootstrap endpoint )

Reply:
Wouldn't it be better to add mapping for this in NginX?